OpenEdXMobile/src/main/java/org/edx/mobile/viewModel/CourseDateViewModel.kt

```diff
@@ -45,6 +55,42 @@ class CourseDateViewModel(
     val errorMessage: LiveData<ErrorMessage>
         get() = _errorMessage
 
+    var eventCreateStartTime: Long = 0L
```

Comment:
* Let's create an int const for 0L so it increases the readability of code. 
* Set property should be private to this class.
* We can update the variable to something generic maybe.